doc/source/notebooks/manual/basics/classification.ipynb

```diff
@@ -4,11 +4,299 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Basic GP Classification model\n",
-    "--\n",
+    "# Basic GP Classification model\n",
     "\n",
-    "see classif notebook\n",
-    "\n"
+    "\n",
+    "This script shows how to build a GP classification model using variational inference. We first look at a one-dimensional example, and then show how this can be adapted when the input space is 2D."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
```

Comment:
you are not using `tf`, can be removed :)

Reply:
done